### PR TITLE
Cal 69 styling updates

### PR DESCRIPTION
**Pull request by angepol**
_Closes:_ <#69>

## Summary of changes
Style adjustments
## Testing

- [ ] Unit tests

#### Screenshot of UI (local testing in browser)

> Shortens day and month to three letters
Screen Shot 2022-12-26 at 11 58 28 AM

> Adds indentation to all day event
Screen Shot 2022-12-26 at 12 00 22 PM

### CODE REVIEW

File: ui/src/lib.ts [hunk 1]
```diff
@@ -44,11 +44,13 @@ const modalDateFormat: Function = ({
   if (startDate === endDate && allDay) {
     return `${startDate}`;
   } else if (startDate === endDate) {
-    return `${startDate} ${timeFormat(start)} - ${timeFormat(end)}`;
+    return `${startDate} · ${timeFormat(start)} - ${timeFormat(end)}`;
   } else if (allDay) {
     return `${startDate} - ${endDate}`;
   } else {
-    return `${startDate} ${timeFormat(start)} - ${endDate} ${timeFormat(end)}`;
+    return `${startDate}, ${timeFormat(start)} - ${endDate}, ${timeFormat(
```
Comment (sofita) [inline ui/src/lib.ts:51]: why not make this one a dot too?

**Review by sofita — COMMENTED** [review]
Nice! I know it's not in the ticket officially, but from Sunyata's comments last demo, she asked if we could abbreviate the months to their 3 letter abbreviation. I know google calendar doesn't do that, but they also don't use the day of the week, so their displayed date is shorter. We could also abbreviate the day of the week if we want to keep it in there.

File: ui/src/lib.test.ts [hunk 2]
```diff
@@ -36,7 +36,7 @@ describe("lib functions", () => {
         allDay: false,
       });
       expect(result).toEqual(
-        "Sunday, December 11 06:43 AM - Wednesday, December 14 08:45 AM"
+        "Sunday, December 11, 06:43 AM - Wednesday, December 14, 08:45 AM",
```
[the remark this answers is not in the document]
Comment (Ashlynn223) [reply ui/src/lib.test.ts:39]: We followed how Google calendar does it. The dot is only used to separate a date and a time frame when it is for a single day. I think it reads nicely like this but we could get Sunyata's input on the next demo.

File: ui/src/lib.ts (hunk 1 above)
Comment (Ashlynn223) [reply ui/src/lib.ts:51]: Similar response to the one I left above.

**Comment by Ashlynn223** [comment]
> Nice! I know it's not in the ticket officially, but from Sunyata's comments last demo, she asked if we could abbreviate the months to their 3 letter abbreviation. I know google calendar doesn't do that, but they also don't use the day of the week, so their displayed date is shorter. We could also abbreviate the day of the week if we want to keep it in there.

Yes, we can add that in as well 🙌

**Comment by Ashlynn223** [comment]
> Nice! I know it's not in the ticket officially, but from Sunyata's comments last demo, she asked if we could abbreviate the months to their 3 letter abbreviation. I know google calendar doesn't do that, but they also don't use the day of the week, so their displayed date is shorter. We could also abbreviate the day of the week if we want to keep it in there.

Google calander does abbreviate the month in the mobile version but their modal on the desktop version does not.

**Review by Ashlynn223 — APPROVED** [review]
Good to merge!

**Pull request merged**